app/cafe.py

```diff
@@ -0,0 +1,17 @@
+import datetime
+from app.errors \
```

Comment:
don't use \
Instead, you can use (NotVaccinatedError, 
                               OutdatedVaccineError, 
                               NotWearingMaskError)

Reply:
fix it